src/App.tsx

```diff
@@ -1,61 +1,97 @@
 import './App.scss';
+import { TodoList } from './components/TodoList';
+import { useState } from 'react';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+export interface Todo {
+  id: number;
+  title: string;
+  userId: number;
+  completed: boolean;
+}
 
 export const App = () => {
+  const [title, setTitle] = useState<string>('');
+  const [selectedUserId, setSelectedUserId] = useState<number | null>(null);
+  const [todos, setTodos] = useState(todosFromServer);
+  const [formSubmited, setFormSubmited] = useState<boolean>(false);
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    setFormSubmited(true);
+
+    if (title.trim() === '' || selectedUserId === null) {
+      return;
+    }
```

Comment:
The condition `selectedUserId === null` might not work as expected because the `value` of the select element defaults to `0` when no user is selected. Consider checking for `selectedUserId === 0` instead, or adjust the logic to ensure `null` is used when no user is selected.